src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect, useRef } from 'react';
+import useResizeObserver from 'use-resize-observer';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { OverflowMenuHorizontal20 } from '@carbon/icons-react';
+
+import { Breadcrumb as CarbonBreadcrumb, OverflowMenuItem, OverflowMenu } from '../../index';
+
+const propTypes = {
+  /**
+   * Specify the label for the breadcrumb container
+   */
+  'aria-label': PropTypes.string,
+
+  /**
+   * Pass in the BreadcrumbItem's for your Breadcrumb
+   */
+  children: PropTypes.node,
+
+  /**
+   * Specify an optional className to be applied to the container node
+   */
+  className: PropTypes.string,
+
+  /**
+   * Optional prop to omit the trailing slash for the breadcrumbs
+   */
+  noTrailingSlash: PropTypes.bool,
+
+  /**
+   * Allow for collapse of breadcrumbs into overflow menu when not enough space, this implies
+   * the use of ResizeObserver
+   */
+  hasOverflow: PropTypes.bool,
+};
+
+const defaultProps = {
+  className: null,
+  noTrailingSlash: false,
+  children: null,
+  hasOverflow: false,
+  'aria-label': null,
+};
+
+const Breadcrumb = ({ children, className, hasOverflow, ...other }) => {
+  const breakingWidth = useRef([]);
+  const { ref: breadcrumbRef } = useResizeObserver({
+    useDefaults: false,
+  });
+  const [overflowItems, setOverflowItems] = useState([]);
+  const [afterOverflowItems, setAfterOverflowItems] = useState(children.slice(1));
+
+  /** update breadcrumbs  */
+  useEffect(() => {
+    if (hasOverflow) {
```

Comment:
I was really consfused about this name `hasOverflow`, but I see now it's an indication that the component will actually check for overflows.

Reply:
> In RTL mode it doesn't appear to be rendering correctly.
> 
> I'm trying to understand how this breadcrumb would be used. In a graphite app, the breadcrumb will be rendered as a link but, we need to hook into the onClick of the item and transition the page using an even on the item instead of a url. Is that even possible here?

I will fix RTL defects. Thanks for the catch. 

For the naming I was trying to stay in line with the `has-*` convention for denoting features that the component has like `Table` component's `hasRowSelection`. 

As for the functionality, I needed to pass the href prop from `BreadCrubmItem` to `OverflowItem`. Adding this now.

Reply:
RTL and onClick items should now be addressed.